site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -51,19 +53,29 @@ export const WorkspacesPageView: FC<WorkspacesPageViewProps> = ({ loading, works
         <TableBody>
           {!workspaces && loading && <TableLoader />}
           {workspaces && workspaces.length === 0 && (
-            <TableRow>
-              <TableCell colSpan={999}>
-                <EmptyState
-                  message={Language.emptyMessage}
-                  description={Language.emptyDescription}
-                  cta={
-                    <Link underline="none" component={RouterLink} to="/workspaces/new">
-                      <Button startIcon={<AddCircleOutline />}>{Language.createButton}</Button>
-                    </Link>
-                  }
-                />
-              </TableCell>
-            </TableRow>
+            <>
+              {query === "owner:me" || query === "" ? (
```

Comment:
I'd make these constants and re-use those constants here and in the form menu. Maybe in `utils/workspaces.ts`